tsconfig.json

```diff
@@ -16,11 +16,9 @@
   "include": ["./src", "typings/*.d.ts"],
   "exclude": ["node_modules/*", "dist"],
   "typedocOptions": {
-    "inputFiles": ["./src"],
+    "entryPoints": ["./src"],
     "out": "doc",
-    "mode": "module",
-    "exclude": ["src/**/index.ts", "src/**/__tests__/**/*.ts"],
-    "excludeNotExported": true,
+    "exclude": ["src/**/__tests__/**/*.ts"],
```

Comment:
If we exclude `"src/**/index.ts"`, then the links could not be resolved and we will receive warnings

<img width="1690" alt="image" src="https://user-images.githubusercontent.com/3513494/228876827-ce069d6e-3f2f-4925-afcc-ddd578120e76.png">